explainer.md

```diff
@@ -0,0 +1,147 @@
+# Open Screen Protocol Explainer
+
+The Open Screen Protocol is a standard way for two agents to discover each other
+on the local network, create a secure network connection, and allow an
+application on one agent to render media on the other.
+
+This capability allows users to discover media on one device (such as a laptop
+or smartphone) and play it on another device (such as a smart TV, set top box,
+or connected speaker).   We call this scenario "second screen" functionality.
+
+## Why should users care?
+
+Currently there are many solutions that allow media accessed on one device to be
+played on a different device.  Examples include [Google
+Cast](https://developers.google.com/cast/), [Apple
+AirPlay](https://www.apple.com/airplay/), and [Spotify
+Connect](https://www.spotify.com/us/connect/).
+
+However, the application, browser and/or OS largely determine what devices work
+together for media playback.  Users can't be easily guaranteed that the content
```

Comment:
While there is no guarantee currently, isn't providing a guarantee specifically a non goal below?